packages/babel-plugin-minify-simplify/src/index.js

```diff
@@ -28,6 +28,62 @@ module.exports = ({ types: t }) => {
   const or = (a, b) => t.logicalExpression("||", a, b);
   const and = (a, b) => t.logicalExpression("&&", a, b);
 
+  const operators = new Set([
+    "+", "-", "*", "%",
+    "<<", ">>", ">>>",
+    "&", "|", "^", "/",
+    "**"
+  ]);
+
+  const updateOperators = new Set([
+    "+", "-"
+  ]);
+
+  function isEqual(arr1, arr2) {
```

Comment:
this is specific to this feature, we should have a comment or change name to indicate that, since this is at the top level (nearly).